bot/exts/evergreen/profile_pic_modification/_effects.py

```diff
@@ -0,0 +1,127 @@
+import typing as t
+from io import BytesIO
+from pathlib import Path
+
+import discord
+from PIL import Image, ImageDraw, ImageOps
+
+EASTER_COLOURS = [
+    (255, 247, 0), (255, 255, 224), (0, 255, 127), (189, 252, 201), (255, 192, 203),
+    (255, 160, 122), (181, 115, 220), (221, 160, 221), (200, 162, 200), (238, 130, 238),
+    (135, 206, 235), (0, 204, 204), (64, 224, 208)
+]  # Pastel colours - Easter-like
+
+
+class PfpEffects():
+    """
+    Implements various image effects.
+
+    All of these methods are blocking, so should be ran in threads.
+    """
+
+    @staticmethod
+    def apply_effect(image_bytes: bytes, effect: t.Callable, *args) -> discord.File:
+        """Applies the given effect to the image passed to it."""
+        im = Image.open(BytesIO(image_bytes))
+        im = im.convert("RGBA")
+        im = effect(im, *args)
+
+        bufferedio = BytesIO()
+        im.save(bufferedio, format="PNG")
+        bufferedio.seek(0)
+
+        return discord.File(bufferedio, filename="modified_avatar.png")
+
+    @staticmethod
+    def closest(x: t.Tuple[int, int, int]) -> t.Tuple[int, int, int]:
+        """
+        Finds the closest easter colour to a given pixel.
+
+        Returns a merge between the original colour and the closest colour
+        """
+        r1, g1, b1 = x
+
+        def distance(point: t.Tuple[int, int, int]) -> t.Tuple[int, int, int]:
+            """Finds the difference between a pastel colour and the original pixel colour."""
+            r2, g2, b2 = point
+            return ((r1 - r2)**2 + (g1 - g2)**2 + (b1 - b2)**2)
```

Comment:
Remove redundant parentheses
```suggestion
            return (r1 - r2) ** 2 + (g1 - g2) ** 2 + (b1 - b2) ** 2
```